### ISSUE TITLE
Unable to locate element

### ISSUE THREAD

**Issue by TEBandCo** [cubicbyte/reddit-account-generator#27]
Hi,

I'm getting the following error when trying to create accounts, just seems to consistently present the bellow error:

```
2024-03-10 17:28:20 INFO Creating reddit account
2024-03-10 17:28:21 DEBUG Opening registration page
2024-03-10 17:28:22 DEBUG Entering email
2024-03-10 17:28:22 DEBUG Quitting driver
2024-03-10 17:28:24 ERROR Message: no such element: Unable to locate element: {"method":"css selector","selector":"[id="regEmail"]"}
  (Session info: chrome=122.0.6261.112)
Stacktrace:
        GetHandleVerifier [0x00007FF7861CAD32+56930]
        (No symbol) [0x00007FF78613F632]
        (No symbol) [0x00007FF785FF42E5]
        (No symbol) [0x00007FF7860398ED]
        (No symbol) [0x00007FF786039A2C]
        (No symbol) [0x00007FF78607A967]
        (No symbol) [0x00007FF78605BCDF]
        (No symbol) [0x00007FF7860781E2]
        (No symbol) [0x00007FF78605BA43]
        (No symbol) [0x00007FF78602D438]
        (No symbol) [0x00007FF78602E4D1]
        GetHandleVerifier [0x00007FF786546ABD+3709933]
        GetHandleVerifier [0x00007FF78659FFFD+4075821]
        GetHandleVerifier [0x00007FF78659818F+4043455]
        GetHandleVerifier [0x00007FF786269766+706710]
        (No symbol) [0x00007FF78614B90F]
        (No symbol) [0x00007FF786146AF4]
        (No symbol) [0x00007FF786146C4C]
        (No symbol) [0x00007FF786136904]
        BaseThreadInitThunk [0x00007FFDCAD67344+20]
        RtlUserThreadStart [0x00007FFDCB0826B1+33]

2024-03-10 17:28:24 ERROR An error occurred during account creation. Trying again...
```

**Comment by IqBroly** [comment]
I am also getting this error. It only happens when the browser opens reddit website in a different themes, it usually used to open that theme rarely but now it only opens that dark theme and can't find the element. Change the email input element to that dark themes and try again

**Comment by IqBroly** [comment]
Remember that it's inside a shadow-root in this theme

**Comment by cubicbyte** [comment]
Guys, it looks like as reddit releases new designs, my bot is becoming useless

It needs to be updated to the new version of reddit

**Comment by sudo-Mystic** [comment]
they are getting stricter , 200+ of mine got suspended

**Comment by sergejbog** [comment]
I have opened a PR #28 that works with their new shadow DOM.

**Comment by cubicbyte** [comment]
Fixed by #28
